### PR TITLE
docs(navbar): Navbar custom title #2082

### LINKED ISSUE #2082: Navigation Bar component Documentation Update
Update Navigation Bar component documentation with the changes of https://github.com/IgniteUI/igniteui-angular/issues/7377

### PR DESCRIPTION
**Pull request by astaev**
Closes #2082

### Checklist:
 
 - [x] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [x] at the References section at the end of the topic add links to topics, samples, etc
 - [x] reference API documentation instead of adding a section with API

<br />

 - [ ] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [ ] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [ ] add inline `code blocks` for the names of classes / tags / properties
 - [ ] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [x] check if sample is working and fully visible in the topic
 - [x] check if sample is working and fully visible in the StackBlitz
 - [x] check if code blocks match the code in StackBlitz demo

<br />

 - [ ] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

File: en/components/navbar.md [hunk 1]
```diff
@@ -126,14 +126,14 @@ What if we want to use a custom action icon for our app navigation instead of th
...
+    <igx-navbar-action>
         <igx-icon class="customIcon" fontSet="fa" name="fa-home"></igx-icon>
-    </igx-action-icon>
+    </igx-navbar-action>
 </igx-navbar>
 ```
 
 > [!NOTE]
-> If [`igx-action-icon`]({environment:angularApiUrl}/classes/igxactionicondirective.html) is provided, the default [`actionButtonIcon`]({environment:angularApiUrl}/classes/igxnavbarcomponent.html#actionbuttonicon) will not be used. 
+> If [`igx-navbar-action`]({environment:angularApiUrl}/classes/igxnavbaractiondirective.html) or [`igxNavbarAction`]({environment:angularApiUrl}/classes/igxnavbaractiondirective.html) is provided, the default [`actionButtonIcon`]({environment:angularApiUrl}/classes/igxnavbarcomponent.html#actionbuttonicon) will not be used.
```
Comment (igdmdimitrov) [inline en/components/navbar.md:136]: Currently these links are not working

File: en/components/navbar.md [hunk 2]
```diff
@@ -188,6 +188,43 @@ If the sample is configured properly, you should see the following in your brows
...
+
+        <igx-icon>search</igx-icon>
+        <igx-icon>favorite</igx-icon>
+        <igx-icon>more_vert</igx-icon>
+    </igx-navbar>
+</div>
+```
+
+> [!NOTE]
+> If [`igx-navbar-title`]({environment:angularApiUrl}/classes/igxnavbartitledirective.html) or [`igxNavbarTitle`]({environment:angularApiUrl}/classes/igxnavbartitledirective.html) is provided, the default [`title`]({environment:angularApiUrl}/classes/igxnavbarcomponent.html#title) will not be used.
```
Comment (igdmdimitrov) [inline en/components/navbar.md:216]: Currently these links are not working

File: en/components/navbar.md [hunk 3]
```diff
@@ -188,6 +188,43 @@ If the sample is configured properly, you should see the following in your brows
 </div>
 <div class="divider--half"></div>
 
+#### Add Custom Title
+
+If we want to provide a custom content for a navbar's title, we can achieve this by using `igx-navbar-title` or `igxNavbarTitle` directives. They will replace the default navbar's title provided by `title` input property. The sample below has a custom title with div and img nodes inside:
```
Comment (DiyanDimitrov) [inline en/components/navbar.md:193]: Technically these are two selectors of the same directive.

```suggestion
If we want to provide a custom content for a navbar's title, we can achieve this by using `igx-navbar-title` or `igxNavbarTitle` directive. They will replace the default navbar's title provided by `title` input property. The sample below has a custom title with div and img nodes inside:
```

Comment (astaev) [reply en/components/navbar.md]: Fixed

**Pull request merged**